### PR TITLE
New package: py-fastai

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-fastai/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    depends_on('py-packaging', type='build')
+    depends_on('py-fastdownload@0.0.5:1', type=('build', 'run'))
+    depends_on('py-fastcore@1.3.22:1.3', type=('build', 'run'))
+    depends_on('py-torchvision@0.8.2:', type=('build', 'run'))
+    depends_on('py-matplotlib', type=('build', 'run'))
+    depends_on('py-pandas', type=('build', 'run'))
+    depends_on('py-requests', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('build', 'run'))
+    depends_on('py-fastprogress@0.2.4:', type=('build', 'run'))
+    depends_on('py-pillow@6.0.1:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-fastai/package.py]:
```suggestion
    depends_on('pil@6.0.1:', type=('build', 'run'))
```
We have a virtual dep for this

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-fastai/package.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-fastai/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-fastai/package.py:1]: Don't do this, it will cause the tests to fail.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/py-fastai/package.py:1]: Apologies.  I didn't realize we were checking that.

File: var/spack/repos/builtin/packages/py-fastai/package.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    fast.ai documentation on Using Colab for more
+    information."""
+
+    homepage = "https://github.com/fastai/fastai/tree/master/"
+    pypi     = "fastai/fastai-2.5.3.tar.gz"
+
+    version('2.5.3', sha256='0cae50617979b052f0ed7337800e6814ee346b792203cf48305709c935e8eeb7')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-fastai/package.py]:
```suggestion
    depends_on('py-setuptools@36.2:', type='build')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**